app/models/book.rb

```diff
@@ -18,4 +18,12 @@ def total_reviews
   def average_rating
     reviews.average(:rating)
   end
+
+  def self.best_books
```

Comment:
looks great, im not going to request a change right now because its so minor, but lets move some of the chaining downwards for readability.
.joins(blah blah)
.group(id)
.order(whatever)